tests/basic.js

```diff
@@ -41,12 +41,40 @@ function testDetail(t, d, uniqueId) {
 
     if (type === Provider.ItemType.MOVIE) {
         t.ok(d.trailer, 'we have a trailer');
+
         t.ok(d.torrents, 'we have a torrents field');
+        var quality = Object.keys(d.torrents)[0];
+        t.ok(quality===Provider.QualityType.DEFAULT
+            || quality===Provider.QualityType.LOW
+            || quality===Provider.QualityType.MEDIUM
+            || quality===Provider.QualityType.HIGH,
```

Comment:
you can use a `in` operator, so that when we change the qualities we don't have to rewrite the tests.